docs/pages/blog/mui-v5.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Announcing MUI Core v5.0
+description: MUI Core v5.0 is out 🎉
+date: 2021-09-15T00:00:00.000Z
+authors: ['mbrookes', 'eps1lon', 'mnajdova', 'michaldudak', 'siriwatknp']
+card: true
+---
+
+After over 400 days of development and over 40 canary releases, we are excited to introduce MUI Core v5.0.0!
+This release features some major highlights:
+
+- ? depending on timing and strategy [A new branding](#a-new-branding)
+- [High-level goals for v5](#high-level-goals-for-v5)
+- [Improved customizability](#improved-customizability)
+- [New components](#new-components)
+- [A new product line: X](#a-new-product-line-x)
+- [Improved DX](#improved-dx)
+- [v4 migration](#v4-migration)
+- [Design kits](#design-kits)
+- [What's next?](#whats-next)
```

Comment:
```suggestion
- [High-level goals for v5](#high-level-goals-for-v5)
- [Improved customizability](#improved-customizability)
- [Improved DX](#improved-dx)
- [A new product line: X](#a-new-product-line-x)
- [New components](#27422 
- [A new branding](#a-new-branding)
- [v4 migration](#v4-migration)
- [Design kits](#design-kits)
- [What's next?](#whats-next)
```

How about that? My rationale is: if v5.0 is mostly about customizability and DX, we should talk about them first. They are the most critical upgrade to the library. Second after would be the introduction of a new product, which is also very critical and important, expanding the company horizons. Branding/design revamp is definitely important but they are mostly to support and promote these first two big changes and to open ground for the next cycle. And as we could have a specific blog about the subject, we could downscale its place on the main v5 blog post TOC. Other topics, as I feel like they're more secondary, could come last.

Reply:
However, moved the rebranding header at the top so that the terminology we use in the content makes sense. Otherwise it's confusing